includes/components/class-attribute.php

```diff
@@ -88,6 +91,42 @@ protected function boot() {
 		}
 	}
 
+    /**
+     * Upgrades attributes.
+     *
+     * @param string $version Old version.
+     * @deprecated Since version 1.7.3
+     */
+    public function upgrade_attributes( $version ) {
+        if ( version_compare( $version, '1.7.3', '<' ) ) {
```

Comment:
If the whole function depends on a condition please use a guard clause instead, there's no need to indent the whole function & prevents a possible accidental adding of code outside of the condition at the bottom